app/src/main/java/com/sun_asterisk/moviedb_50/screen/home/HomeFragment.kt

```diff
@@ -82,10 +82,10 @@ class HomeFragment : Fragment(),
     override fun onLoading(isLoad: Boolean) {
         view?.run {
             if (!isLoad) {
```

Comment:
đơn giản biểu thức check 

```suggestion
            if (isLoad) {
```